recipe/meta.yaml

```diff
@@ -15,37 +15,39 @@ source:
         - make_tests_osx.patch  # [osx]
         # modify CMakeLists to use OpenBLAS rather than Accelrate on OS X
         # and to use OpenBLAS as the LAPACK library on linux
-        - CMakeLists.patch
+        - CMakeLists.patch  # [unix]
 
 build:
-  number: 201
-  skip: true  # [win]
+  number: 202
+  skip: true  # [win32]
   features:
     - blas_{{ variant }}
 
 requirements:
   build:
     - cmake
+    - toolchain
```

Comment:
Use `{{ compiler('cxx') }}`

Reply:
I testes before with ```{{ compiler('c') }}``` and ```{{ compiler('cxx') }}``` but it didn't work. However I will try it again
Do you think I have to replace ```toolchain``` by ```{{ compiler('c') }}``` ?